### DISCUSSION TITLE
'enable module' is greyed out

### DISCUSSION THREAD

**Discussion by jeromekleinen** [LSPosed/LSPosed#2305] [Q&A]
Hello,

I am trying to create a simple lsposed module. I am just following some guides like https://blog.csdn.net/scrawman/article/details/127187151 to create a simple hello world example. I can compile and install the example module and lsposed manager sees the module, but the 'enable module' slider button is greyed out and I cannot figure out why it won't let me enable the module.

Edit: after posting I did come across a similar question here https://github.com/orgs/LSPosed/discussions/2045 but their example was crashing the daemon. I am only trying to log the loading of the module using XposedBridge for now. I see no entries in the logs, also not the verbose logs.

Edit 2: for completeness sake I downloaded the source code of another module, compiled it and installed it on the device and this I can enable just fine.

Thanks,

Jerome

**Comment by mywalkb** [comment]
you should post your code for understand something or at least post debug logs.
If another module works and you have source code, start to develope from it, I think is better.
